ddtrace/tracer/option.go

```diff
@@ -443,6 +449,24 @@ func WithHostname(name string) StartOption {
 	}
 }
 
+// WithHTTPClientErrorStatuses specifies the range of HTTP client status codes that are marked as errors.
```

Comment:
We should specify the format we expect `codes` to be in. Is this a comma-separated list?

Reply:
Why not `...int`? That way, a user could do:
```go
tracer.WithHTTPClientErrorStatuses(404, 405, 403)
```
or even
```go
tracer.WithHTTPClientErrorStatuses(http.StatusNotFound, http.StatusError)
```

Reply:
I thought we wanted to represent them in string form as comma-separated ranges? Ex: "400-405,410"

Reply:
Right, we can do that, and be in line with the values for the env. vars. In that case, let's clearly describe in the documentation what the `string` argument can be...